src/ui.message.js

```diff
@@ -78,15 +75,18 @@ var _ = Mavo.UI.Message = $.Class({
 	},
 
 	close: function(resolve) {
-		$.transition(this.element, {opacity: 0}).then(() => {
+		if (this.o.closeTransition) {
+			$.transition(this.element, {opacity: 0}).then(() => {
+				$.remove(this.element);
+				this.closed.resolve(resolve);
+			});
+		}
+		else {
 			$.remove(this.element);
 			this.closed.resolve(resolve);
-		});
```

Comment:
I’m not a huge fan of this, we are repeating the closing code twice. Right now it's only 2 lines, but in the future it could be more, and just like all WET code, it means that the repeated snippets can easily get out of sync by someone making changes hastily.
Did setting the duration to 0 not work?
In that case, I'd suggest using `Promise.resolve()` instead of `$.transition()` and then the `.then()` part is the same.
Also, you don't need `closeTransition` if I'm reading this right, just reading the computed style for a transitionDuration should work (but I may be missing something, I'm reviewing this in a rush at the airport right before boarding).